src/contexts/Balances/index.tsx

```diff
@@ -139,6 +130,18 @@ export const BalancesProvider = ({
     };
   }, []);
 
+  // subscribe to added accounts
+  const handleSubscribe = async (accountsAdded: Array<ImportedAccount>) => {
+    // subscribe to balances
+    Promise.all(
```

Comment:
Missing await on Promise.all ?

Reply:
I opted not to await, so the ledgers and balances can both subscribe concurrently. They don't depend on each other so in the interest of speed I opted not to await.